### PR TITLE
fix: change copy on home page

### LINKED ISSUE #7570: Fit & finish - UI Copy | Home page
Fit & finish - UI Copy | Home page

White cells = existing copy
Yellow/blue cells = new copy
Orange = P0

Link to 'live' copy on 'Home page' tab: 
https://microsoft.sharepoint.com/:x:/r/teams/ConversationalAI785/_layouts/15/Doc.aspx?sourcedoc=%7B6B1FACBD-B5FE-4AD2-A35B-995635C1696B%7D&file=glossary.xlsx&action=edit&mobileredirect=true&wdPreviousSession=36539654-a42b-4b87-be1b-de96507b4422&wdOrigin=TEAMS-ELECTRON.teams.undefined&cid=676cf454-61e0-4182-bd27-39044bb51cea

Homepage_copy
Home_Tutorials and videos
Home_whats new

### PR DESCRIPTION
**Pull request by beyackle**
## Description

This fixes the home page to include the new copy changes. The ones inside the `feed.json` file are impossible to screen-capture until this merges; those changes show up in the "What's New" sidebar and the Videos/Tutorials section at the bottom and are read dynamically out of this very repository on demand.

## Task Item

closes #7570

## Screenshots

image

### CODE REVIEW

File: feed.json [hunk 1]
```diff
@@ -2,48 +2,48 @@
...
-			"title": "Microsoft AI Show - Building Telephony solutions with Composer",
-			"description": "A look at the new Azure Bot Service Telephony channel preview and Composer updates focused on Speech",
+			"title": "AI Show: Building a Telephony bot",
+			"description": "How to enable telephony with Composer",
 			"url": "https://channel9.msdn.com/Events/All-Around-Azure/All-Around-Azure-A-Developers-Guide-to-AI/Improve-Customer-Engagement-and-Productivity-with-Conversational-AI"
 		}, {
 			"image": "https://github.com/microsoft/BotFramework-Composer/raw/main/docs/home-feed-content/all-around-azure.png",
-			"title": "Conversational AI - All Around Azure",
-			"description": "Part of the All Around Azure event. A overview of Conversational AI and a walktrhough of building a bot with Bot Framework Composer.",
+			"title": "All Around Azure: Conversational AL",
```
Comment (tdurnford) [inline feed.json:35]:
```suggestion
			"title": "All Around Azure: Conversational AI",
```

**Review by tdurnford — CHANGES_REQUESTED** [review]

File: feed.json [hunk 2]
```diff
@@ -2,48 +2,48 @@
...
 	}, {
 		"title": "Build 2020 updates",
-		"description": "Read all of the updates from this years Build conference, including Composer GA and SDK v4.9.",
+		"description": "Read all of the updates from the Build conference, including Composer GA and Bot Framework SDK v4.9.",
 		"url": "https://techcommunity.microsoft.com/t5/azure-ai/build-2020-conversational-ai-updates/ba-p/1397685"
 	}, {
-		"title": "Power Virtual Agents and Composer integration preview",
-		"description": "We are pleased to announce the Public Preview of Power Virtual Agents integration with Microsoft Bot Framework Composer. Bot makers will now be able to use Bot Framework Composer to create custom content and add it to Power Virtual Agents.",
+		"title": "Microsoft Ignite 2020: Publish to Microsoft Power Virtual Agents ",
+		"description": "Open a Power Virtual Agents topic in Bot Framework Composer, create a custom bot and publish back to Microsoft Power Virtual Agents.",
```
Comment (tdurnford) [inline feed.json:13]: Oxford comma
```suggestion
		"description": "Open a Power Virtual Agents topic in Bot Framework Composer, create a custom bot, and publish back to Microsoft Power Virtual Agents.",
```

File: feed.json [hunk 3]
```diff
@@ -2,48 +2,48 @@
...
 		"url": "https://powervirtualagents.microsoft.com/en-us/blog/power-virtual-agents-integration-with-bot-framework-composer-is-available-in-public-preview/"
 	}, {
-		"title": "November 2020 - Conversational AI update",
-		"description": "Updates to the Conversational AI platform, including a new release of Bot Framework Composer and Power Virtual Agents integration.",
+		"title": "Microsoft Ignite 2020: Conversational AI",
```
Comment (tdurnford) [inline feed.json:16]: Just want to double check that the year should be 2020 and not 2021

File: feed.json [hunk 4]
```diff
@@ -2,48 +2,48 @@
...
+		"description": "Updates to Conversational AI include Microsoft Teams APIs, and new releases for .NET, JavaScript, Python and Java (preview 7).",
```
Comment (tdurnford) [inline feed.json:17]: Unnecessary comma 
```suggestion
		"description": "Updates to Conversational AI include Microsoft Teams APIs and new releases for .NET, JavaScript, Python and Java (preview 7).",
```

File: feed.json [hunk 5]
```diff
@@ -2,48 +2,48 @@
...
 		"url": "https://techcommunity.microsoft.com/t5/azure-ai/november-2020-conversational-ai-update/ba-p/1892528"
 	}, {
-		"title": "Conversational AI at Microsoft Ignite 2020",
-		"description": "Conversational AI updates from Microsoft Ignite 2020.",
+		"title": "Microsoft Ignite 2020: Integration of QnA into Composer",
```
Comment (tdurnford) [inline feed.json:20]: Again, just want to double check the year

File: feed.json [hunk 6]
```diff
@@ -2,48 +2,48 @@
...
 	"tabs": [{
 		"title": "Latest Videos",
-		"viewAllLinkText": "Visit the Bot Framework Composer YouTube Channel",
+		"viewAllLinkText": "View all",
 		"viewAllLinkUrl": "https://www.youtube.com/channel/UC8qPRh20PpuxBvxf0w2WrwQ",
 		"cards": [{
 			"image": "https://github.com/microsoft/BotFramework-Composer/raw/main/docs/home-feed-content/ai-show.png",
-			"title": "Microsoft AI Show - Building Telephony solutions with Composer",
-			"description": "A look at the new Azure Bot Service Telephony channel preview and Composer updates focused on Speech",
+			"title": "AI Show: Building a Telephony bot",
```
Comment (tdurnford) [inline feed.json:30]: Should 'Telephony' be lowercase?

File: feed.json [hunk 7]
```diff
@@ -52,23 +52,23 @@
 		"viewAllLinkUrl": "https://www.youtube.com/channel/UC8qPRh20PpuxBvxf0w2WrwQ",
 		"cards": [{
 			"image": "https://github.com/microsoft/BotFramework-Composer/raw/main/docs/home-feed-content/composer-tutorials.png",
-			"title": "Using Prompts",
-			"description": "Walkthrough and tips for using Prompts to accept information from users.",
+			"title": "Collect user input",
+			"description": "How to use 'Ask a question' actions to collect text, number or other input.",
```
Comment (tdurnford) [inline feed.json:56]: Oxford comma 🤷‍♂️
```suggestion
			"description": "How to use 'Ask a question' actions to collect text, number, or other input.",
```

File: feed.json [hunk 8]
```diff
@@ -52,23 +52,23 @@
...
 			"url": "https://www.youtube.com/watch?v=dUT8L4_bVrk"
 		}, {
 			"image": "https://github.com/microsoft/BotFramework-Composer/raw/main/docs/home-feed-content/composer-tutorials.png",
-			"title": "LG for Advanced Responses",
-			"description": "How to use LG (Language Generation) to create dynamic responses.",
+			"title": "Advanced bot responses",
+			"description": "How to create dynamic, condition based bot responses.",
```
Comment (tdurnford) [inline feed.json:61]: 🤷‍♂️
```suggestion
			"description": "How to create dynamic, conditional based bot responses.",
```

File: feed.json (hunk 1 above)
Comment (beyackle) [reply feed.json:35]: Typo fixed.

File: feed.json (hunk 3 above)
Comment (beyackle) [reply feed.json:16]: Yes, that's where that link goes (November 2021 is still a ways off).

File: feed.json (hunk 5 above)
Comment (beyackle) [reply feed.json:20]: Same here.

**Review by tdurnford — APPROVED** [review]

**Pull request merged**